### DISCUSSION TITLE
Bagaimana cara mengubah format angka pada view, misal: 1000000 jadi 1,000,000?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#220] [Q&A]
Kang, bagaimana cara memformat angka ribuan, ratusan ribu, jutaan, milyaran terutama pada angka view?

ataupun memformat dalam bentuk 1K, 1M, 1B

**Comment by taufik-nurrohman** [comment]
~~~ php
Hook::set('page.view', function($content) {
    if ($content) {
        // <https://www.php.net/number_format>
        return number_format($content);
    }
    return $content;
}, 0);
~~~

Tapi tetap harus hati-hati ya karena sekarang keluaran angka sudah bukan lagi `int` melainkan `string`. Beberapa fitur lain yang tergantung dengan angka ini mungkin tidak bisa bekerja lagi.

> ataupun memformat dalam bentuk 1K, 1M, 1B

<https://stackoverflow.com/a/4703481/1163000>

**Reply by yuliakbard** [reply]
gak bisa kang, error
